vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -35,20 +37,28 @@ import (
 	"github.com/onsi/gomega"
 )
 
+func init() {
+	// Dynamically register feature gates from the VPA's versioned feature gate configuration
+	// This ensures consistency with the main VPA feature gate definitions
+	if err := utilfeature.DefaultMutableFeatureGate.Add(features.MutableFeatureGate.GetAll()); err != nil {
+		panic(fmt.Sprintf("Failed to add VPA feature gates: %v", err))
+	}
+}
+
 const (
 	webhookConfigName = "vpa-webhook-config"
 	webhookName       = "vpa.k8s.io"
 )
 
-var _ = AdmissionControllerE2eDescribe("Admission-controller", ginkgo.Label("FG:InPlaceOrRecreate"), func() {
```

Comment:
Are these labels just removed then? Does the `WithFeatureGate` function add something like this for us?

Reply:
Correct, it adds these:
```
[FeatureGate:PerVPAConfig] [Alpha] [Feature:OffByDefault]
```

For beta it just adds the FeatureGate and Beta, it doesn't add an "[Feature:OnByDefault]"